### ISSUE TITLE
How can we show conversion metrics for variants that are not comparable

### ISSUE THREAD

**Issue by kingo55** [mint-metrics/mojito-r-analytics#4]
We often run experiments that implement  a new feature that does not exist on the control group.

We want to see how many users are interacting with a particular feature, however we can't compare it against the control. This leads us to data like:

 - Control: 0% conversion
 - Treatment: 10% conversion (Infinite increase over the control group)

This makes our summary table hard to read and the metric plots somewhat misleading.

**Comment by dapperdrop** [comment]
Agreed, we need to treat these conversion points (goals) separately. 

In the summary section, we can have another table, e.g. :

Goal | Recipe | Cvr
--- | --- | ---
Some Goal | Treatment | 10%

If there are multiple treatments, perhaps we can conditionally format the cvr column to highlight the best performers.

Maybe the best way to flag these conversion points is by another attribute in `goalList`?

e.g.
```
list(
    title="Some Goal",
    goal="some_goal",
    operand="like",
    noControl = true
  )
```
